docs/en/samples/automatic_clean_up_data_operation.md

```diff
@@ -0,0 +1,112 @@
+# Demo - Automatic Cleanup Data Operation
+
+## Background
+
+Fluid's universal data operations describe operations such as data prefetch, data migration, elastic scaling, cache cleaning, metadata backup, and recovery.
+Similar to the Kubernetes Job's automatic cleaning mechanism, we also provides automatic cleanup data operation, utilizing the Time-to-Live (TTL) mechanism to limit the lifecycle of data operations that have finished execution. This document will briefly demonstrate the utilization of these features.
```

Comment:
In the "Background" section, I suggest replacing "we also provide" with "Fluid also provides", and using "leveraging" instead of "utilizing" to make the statement sound more natural.